src/storage/backup/src/lib.rs

```diff
@@ -58,6 +60,8 @@ pub struct MetaSnapshotMetadata {
     #[serde(default)]
     pub format_version: u32,
     pub remarks: Option<String>,
+    #[serde(with = "table_id_key_map")]
```

Comment:
If we set key type as string or u32, we may not need to impl the serde by ourselves?

Reply:
serde_json requires map key to be String. So either using String as key or impl the serde.

The downside of using String as key is additional type conversion when visiting the map.